src/test/Components/ImagesTable/ImagesTable.test.tsx

```diff
@@ -127,7 +129,9 @@ describe('Images Table', () => {
 
     // No actual clicking because downloading is hard to test.
     // Instead, we just check href and download properties of the <a> element.
-    const downloadLink = await within(downloadButton).findByRole('link');
+    const downloadLink = (await within(downloadButton).findByRole(
+      'link'
+    )) as HTMLAnchorElement;
```

Comment:
Again, let's use `const downloadLink: HTMLAnchorElement` instead of `as` please.

Reply:
... done...